db/seeds.rb

```diff
@@ -6,23 +6,24 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 CompanyValue.destroy_all
+Kudo.destroy_all
```

Comment:
seeds are used to populate a clean database, that's why you should make the assumption that db is empty and you shouldn't need to destroy anything at the beginning.

If you want to replant db just use the command `rails db:seed:replant`